ci/kind/kind-setup.sh

```diff
@@ -538,5 +576,9 @@ if version_lt "$kind_version" "0.12.0" && [[ "$KUBE_PROXY_MODE" == "none" ]]; th
 fi
 
 if [[ $ACTION == "create" ]]; then
-      create
+    if [[ ! -z $SUBNETS ]] && [[ ! -z $EXTRA_NETWORKS ]]; then
+        echoerr "Only one of --subnets and --extra-networks can be specified"
```

Comment:
nit: could you surround the flags with single quotes in error messages?

Reply:
Done